tools/gke-gpu-driver-version/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "gke-gpu-driver-version"
+version = "0.1.0"
+edition = "2024"
```

Comment:
Because this is a binary crate, I would recommend checking in the `Cargo.lock` as well. That should make issues more reproducible (and help avoid fringe issues to begin with).